.github/workflows/sanity-ssl.yaml

```diff
@@ -0,0 +1,20 @@
+name: Build & Sanity SSL
+on: [push, pull_request]
+
+jobs:
+  run-sanity-ssl-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Run Build & SSL Sanity Tests
+        run: |
+          set -x
+          mkdir -p logs/sanity-test-logs
+          chmod 777 logs/sanity-test-logs
+          make test-external-pg-sanity
```

Comment:
why do we need a different workflow? can't we use the sanity workflow and run both make test-external-pg-sanity & make test-sanity

Reply:
Decided to leave for now only test-external-pg-sanity. The unit test will run only on default Postgres for now.